### PR TITLE
RK-9473 - snyk may 21

### PR DESCRIPTION
**Pull request by niryarden**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -132,7 +132,7 @@
     "vscode-languageserver-protocol": "3.15.3",
     "vscode-ws-jsonrpc": "^0.2.0",
     "walk": "^2.3.13",
-    "ws": "^5.0.0"
```
Comment (dan505512) [inline package.json:135]: Just bump up the explorook version

Comment (niryarden) [reply package.json:135]: no need because we didn't publish the last version yet.

**Comment by niryarden** [comment]
@dan505512 
IDE nav is broken in this branch
i'm looking at it

no need for review right now

File: package.json (hunk 1 above)
Comment (dan505512) [reply package.json:135]: https://github.com/rookout/explorook/releases/latest

**Comment by niryarden** [comment]
https://github.com/Rookout/explorook/pull/267

**Pull request closed without merging**